src/Microsoft.AspNet.Hosting/HostingServicesCollectionExtensions.cs

```diff
@@ -17,6 +17,7 @@ public static IServiceCollection AddLogging(this IServiceCollection services, IC
         {
             var describe = new ServiceDescriber(config);
             services.TryAdd(describe.Singleton<ILoggerFactory, LoggerFactory>());
+            services.TryAdd(describe.Singleton(typeof(ILogger<>), typeof(Logger<>)));
```

Comment:
@davidfowl are you sure this should be a singleton? `ILogger` and `Logger` in particular don't appear to be thread safe.